lib/common/strings.c

```diff
@@ -111,6 +111,39 @@ pcmk__scan_ll(const char *text, long long *result, long long default_value)
     return rc;
 }
 
+/*!
+ * \internal
+ * \brief Scan an integer value from a string, constrained to a minimum
+ *
+ * \param[in]  text           The string to scan (may be NULL)
+ * \param[out] result         Where to store result (or NULL to ignore)
+ * \param[in]  minimum        Value to use as default and minimum
+ *
+ * \return Standard Pacemaker return code
+ */
+int
+pcmk__scan_min_int(const char *text, int *result, int minimum)
```

Comment:
Basically, I think any new function named pcmk__something should get unit tests.